src/main/resources/langs/fr_FR.yml

```diff
@@ -389,6 +389,11 @@ fun:
         mustIncludeTopAndBottomText: "Tu dois inclure les arguments `top text` et `bottom text` pour générer un meme."
         listItem: "`{0}` => `{1}`"
 
+
+    PunCommand:
+     title: "Pun list"
```

Comment:
The `title` and `noResults` should be indented by four spaces.

Reply:
Indentation should be correct now.